src/bot_telegram.ts

```diff
@@ -160,13 +161,46 @@ export default class TelegramBot extends BotClient {
         this.logError(`Failed to send message to channel:\n${error}`);
       }
     } else {
-      let text = this.msgFromMarkdown(message.toMDString());
+      const link = message.title.link;
+      let text = '';
+      let templateFound = false;
+
+      const templates = message.game.telegramIVTemplates;
+      this.logDebug(`There are ${templates.length} templates for ${message.game.label}.`);
+
+      // Test for IV template matches
+      for (const telegramIVtemplate of templates) {
```

Comment:
This assumes that multiple matching templates mean the last one overrides any previous. I think it's fine but there's probably a better way to do it than checking all the templates no matter what.

Reply:
Right, in a previous implementation it would return in the loop. Fixed.